.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: c
+cache: apt
+dist: trusty
+addons:
+  apt:
+    sources: &apt_sources
+      - ubuntu-toolchain-r-test
+      - sourceline: 'ppa:kalakris/cmake' # cmake 2.8 for precise
+      - sourceline: 'ppa:adrozdoff/cmake' # cmake 3 for trusty
+      - sourceline: 'ppa:h-rayflood/llvm'
+      - sourceline: ppa:hvr/z3   # z3 for the smt interface package
+    packages: &default_packages
+      - cmake
+      - libgmp-dev
+      - libgoogle-perftools-dev
+      - g++-4.9
+      - clang-3.4
+matrix:
+  include:
+   - os: linux
+     dist: precise
+     env:
+       CMAKE_CXX_COMPILER=g++-4.9
+       CMAKE_BUILD_TYPE=Release
```

Comment:
I believe we do not need `matrix` and the above can be simplified as we only care about one compiler on one platform. (Ubuntu Trusty in this case.) For example we only need `clang` or `g++` but not both.